README.md

```diff
@@ -1 +1,139 @@
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [About the Project] <a name="about-project"></a>
+
+
+**[my portfolio]** is a project that displays the projects that I have previously worked on.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Styling</summary>
+  <ul>
+    <li><a href="https://tailwindcss.com/docs/installation">tailwindcss</a></a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Interactivity</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">JavaScript</a></a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Landing_page]**
+- Landing page of the website
+
+- **[About_page]**
+- This is a page that contains information about the dev summit.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Click [here](http://module1capstone.netlify.app/) to see the live demo of the project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+- Clone the repository
+- Install dependencies
+- To install all dependencied, run `npm install` in your terminal.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. [LiverServer](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer)
+1. [FiveServer] [LiverServer](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer)
+   )
+
+### Install
+
+Install this project with:
+
+```cd module_1_capstone
+npm install
+```
+## 👥 Authors <a name="authors"></a>
+
+👤 **Martin Karugaba**
+
+- GitHub: [@githubhandle](https://github.com/martinkarugaba)
+- Twitter: [@twitterhandle](https://twitter.com/martin_karugaba)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- All icons in the main programs section were got from [flaticon.com](https://www.flaticon.com/)
+
```

Comment:
As requested in the project requirements:  
![image](https://user-images.githubusercontent.com/42799579/203677364-a4f79c4a-ebf5-467a-b9c4-e22103c2cc57.png)  

Please give credit to the [original author](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) of the design 😉.  
> Note: the credits should be in the `Acknowledgments` section 

This is the only required change! Your project looks flawless, well done, and keep up the excellent work 😄!